test/cli/repl/run

```diff
@@ -7,7 +7,7 @@ do
     echo " --- Test: $dir"
     if [ $update_expected -eq 1 ]
     then
-        dist/build/icicle-repl/icicle-repl ":import data/libs/prelude.icicle" ":load test/cli/repl/data.psv" < $dir/script | diff $dir/expected -
+        dist/build/icicle-repl/icicle-repl ":import data/libs/prelude.icicle" ":load test/cli/repl/data.psv" < $dir/script | diff -u $dir/expected -
```

Comment:
Yeah it makes it the same as a `git diff` so you get a bit more context:

**file.a**
```
let x = read# [Int] ptr
let y = read# [Int] ptr

foreach (e : es) {
  printf# "f(x, y, e) = %d" (f x y e)
}

let z = read# [Int] ptr
```

**file.b**
```
let x = read# [Int] ptr
let y = read# [Int] ptr

foreach (e : es) {
  printf# "g(x, y, e) = %d" (g x y e)
}

let z = read# [Double] ptr
```

**diff file.a file.b**
```diff
5c5
<   printf# "f(x, y, e) = %d" (f x y e)
---
>   printf# "g(x, y, e) = %d" (g x y e)
8c8
< let z = read# [Int] ptr
---
> let z = read# [Double] ptr
```

**diff -u file.a file.b**
```diff
--- file.a	2015-08-28 05:11:08.000000000 +1000
+++ file.b	2015-08-28 05:11:12.000000000 +1000
@@ -2,7 +2,7 @@
 let y = read# [Int] ptr

 foreach (e : es) {
-  printf# "f(x, y, e) = %d" (f x y e)
+  printf# "g(x, y, e) = %d" (g x y e)
 }

-let z = read# [Int] ptr
+let z = read# [Double] ptr
```